src/Raven.Server/Monitoring/Snmp/Objects/Server/1.5/ProcessLoadAverage15M.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Lextm.SharpSnmpLib;
+
+namespace Raven.Server.Monitoring.Snmp.Objects.Server
+{
+    public sealed class ProcessLoadAverage15M() : ProcessLoadAverageBase(SnmpOids.Server.ProcessLoadAverage15M, 2)
+    {
+    }
+}
```

Comment:
It's not Process. Load Average is a metric for the entire machine. Let's drop the prefix "Process". Optionally replace it with Machine - MachineLoadAverage

Reply:
is close, all of them are in 1.5.x category. Machine CPU is 1.5.2 and next available slot was 1.5.6.x